distribution/zip/jballerina/bin/bal

```diff
@@ -195,6 +195,11 @@ do
     BALLERINA_CLASSPATH="$BALLERINA_CLASSPATH":$j
 done
 
+for j in "$BALLERINA_HOME"/repo/bala/ballerina/cloud/*/any/compiler-plugin/libs/*.jar
```

Comment:
@hevayo @hemikak @xlight05 
Could this cause any issues when pulling `cloud` from central?